docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -62,6 +64,18 @@ Perhaps. GitHub permissions can quickly get complicated, as it involves two leve
 
 First of all, you can only configure already-installed GitHub Apps for organizations you are not an admin in if you go through the app installation process again. Be aware that in this scenario, you can only add repos that you have admin access to, not just maintainer access.
 
+There are two scenarios to be aware of when installing the Dockstore GitHub App as a repository admin who is not an organization admin.
+
+1. An organization admin configured the Dockstore GitHub App to be installed on all repositories.
+
+   * Attempts to install the Dockstore GitHub App on any repository in the organization will result in an error from GitHub, regardless of whether or not the repository is one that the user has admin access to.
+   * If it is a repository that the user has admin access to, the user can :ref:`check the repository to see if the app is indeed installed <Check GitHub App installation on repository>`.
```

Comment:
This doesn't answer the question about whether all-repo access applies to new repos. What happens in this scenario?
1. Adam Organization-Admin installs app to have universal repo access (ie access to all repos)
2. Bob Repo-Admin later makes a new repo on the organization

Does Bob need to install the app at all or is the new repo already added?

Reply:
![Screenshot from 2022-09-16 10-10-37](https://user-images.githubusercontent.com/1730679/190659061-daf8dd61-fbc9-4759-b5f8-2515347c3b4a.png)

This is more of a GitHub question than a Dockstore question, but GitHub does say that it would apply to future repos